stdpopsim/arabidopsis_thaliana.py

```diff
@@ -0,0 +1,112 @@
+"""
+Genome, genetic map and demographic model definitions for Arabidopsis thaliana.
+"""
+import msprime
+
+import stdpopsim.models as models
+import stdpopsim.genomes as genomes
+import stdpopsim.genetic_maps as genetic_maps
+
+
+###########################################################
+#
+# Genetic maps
+#
+###########################################################
+
+
+class Salome2012(genetic_maps.GeneticMap):
+    """
+    Genetic map from Salome 2012 averaged across population crosses.
+    Please see README for details on how this was done.
+    http://www.eeb.ucla.edu/Faculty/Lohmueller/data/uploads/
+        README_athaliana_map_salome2012.txt
```

Comment:
I'm getting a URL not found, but I'm assuming that this might just be an issue with permission to access UCLA servers.

Reply:
I just tried it from an Oxford university address and it worked fine, but then tried it from my home IP and got a 
```
Failed to establish a new connection: [Errno -2] Name or service not known',
```
Looks like the UCLA server isn't known outside academic circles?

Reply:
I just tried it from an Oxford university address and it worked fine, but then tried it from my home IP and got a 
```
Failed to establish a new connection: [Errno -2] Name or service not known',
```
Looks like the UCLA server isn't known outside academic circles?

Reply:
Err nope, ignore that. Turned it was just my network wasn't on properly. I can download the map fine from my home IP.

Reply:
@jradrion is this working for you now?

Reply:
Sorry, just getting back to looking at this now. No, it still doesn't work for me; must be something on my end. I have no issues accessing "http://www.eeb.ucla.edu/Faculty/Lohmueller/data/uploads/salome2012_maps.tar.gz", however.